sqlmesh/core/model/definition.py

```diff
@@ -698,7 +711,7 @@ def is_python(self) -> bool:
     def is_seed(self) -> bool:
         return False
 
-    @cached_property
+    @property
```

Comment:
Why this changed?

Reply:
because it's annoying to deal with cached_properties in cleanup